packages/database/src/migrations/20210603064215-migrateFijiFacilitiesToNewSubDivisions-modifies-data.js

```diff
@@ -0,0 +1,282 @@
+  FJ_054: 'FJ_sdLautoka_Yasawa', //	Lautoka Hub Centre
+  FJ_238: 'FJ_sdNadroga', //	Loma Nursing Station
+  FJ_089: 'FJ_sdNadroga', //	Lomawai Health Centre
+  FJ_050: 'FJ_sdLautoka_Yasawa', //	Malolo Health Centre
+  FJ_073: 'FJ_sdNadi', //	Momi Health Centre
+  FJ_244: 'FJ_sdNadroga', //	Nabou Nursing Station
+  FJ_051: 'FJ_sdLautoka_Yasawa', //	Nacula Health Centre
+  FJ_082: 'FJ_sdTavua', //	Nadarivatu Health Centre
+  FJ_065: 'FJ_sdNadi', //	Nadi Health Centre
+  FJ_064: 'FJ_sdNadi', //	Nadi Sub-divisional Hospital
+  FJ_084: 'FJ_sdTavua', //	Nadrau Nursing Station
+  FJ_069: 'FJ_sdNadi', //	Nagado Nursing Station
+  FJ_083: 'FJ_sdTavua', //	Nagatagata Nursing Station
+  FJ_076: 'FJ_sdBa', //	Nailaga Health Centre
+  FJ_079: 'FJ_sdBa', //	Nalotawa Nursing Station
+  FJ_067: 'FJ_sdNadi', //	Namaka Health Centre
+  FJ_219: 'FJ_sdNadi', //	Namaka Reproductive Health
+  FJ_104: 'FJ_sdRa', //	Namarai Health Centre
+  FJ_078: 'FJ_sdBa', //	Namau Nursing Station
+  FJ_070: 'FJ_sdNadi', //	Nanoko Nursing Station
+  FJ_103: 'FJ_sdRa', //	Nanukuloa Health Centre
+  FJ_096: 'FJ_sdNadroga', //	Naqalimare Nursing Station
+  FJ_105: 'FJ_sdRa', //	Nasau Health Centre
+  FJ_110: 'FJ_sdRa', //	Nasavu Nursing Station
+  FJ_052: 'FJ_sdLautoka_Yasawa', //	Natabua Health Centre
+  FJ_071: 'FJ_sdNadi', //	Nausori Highland Nursing Station
+  FJ_072: 'FJ_sdNadi', //	Nawaicoba Nursing Station
+  FJ_109: 'FJ_sdRa', //	Nayavuira Nursing Station
+  FJ_097: 'FJ_sdNadroga', //	Nukuilau Nursing Station
+  FJ_062: 'FJ_sdLautoka_Yasawa', //	Punjas Health Centre
+  FJ_236: 'FJ_sdRa', //	Ra Maternity Hospital
+  FJ_090: 'FJ_sdNadroga', //	Raiwaqa Health Centre
+  FJ_102: 'FJ_sdRa', //	Rakiraki Health Centre
+  FJ_100: 'FJ_sdRa', //	Rakiraki Sub-divisional Hospital
+  FJ_053: 'FJ_sdLautoka_Yasawa', //	Sai Veiseisei Health Centre
+  FJ_215: 'FJ_sdNadi', //	Sarada Medical Centre
+  FJ_086: 'FJ_sdNadroga', //	Sigatoka Health Centre
+  FJ_085: 'FJ_sdNadroga', //	Sigatoka Sub-divisional Hospital
+  FJ_060: 'FJ_sdLautoka_Yasawa', //	Somosomo Nursing Station
+  FJ_099: 'FJ_sdNadroga', //	Tau Nursing Station
+  FJ_081: 'FJ_sdTavua', //	Tavua Health Centre
+  FJ_235: 'FJ_sdTavua', //	Tavua Maternity
+  FJ_080: 'FJ_sdTavua', //	Tavua Sub-divisional Hospital
+  FJ_055: 'FJ_sdLautoka_Yasawa', //	Teci Nursing Station
+  FJ_107: 'FJ_sdRa', //	Tokaimalo Nursing Station
+  FJ_098: 'FJ_sdNadroga', //	Tuvu Nursing Station
+  FJ_093: 'FJ_sdNadroga', //	Vatukarasa Health Centre
+  FJ_091: 'FJ_sdNadroga', //	Vatulele Health Centre
+  FJ_058: 'FJ_sdLautoka_Yasawa', //	Viwa Nursing Station
+  FJ_108: 'FJ_sdRa', //	Vunitogoloa Nursing Station
+  FJ_095: 'FJ_sdNadroga', //	Wauosi Nursing Station
+  FJ_056: 'FJ_sdLautoka_Yasawa', //	Yalobi Nursing Station
+  FJ_057: 'FJ_sdLautoka_Yasawa', //	Yanuya Nursing Station
+  FJ_059: 'FJ_sdLautoka_Yasawa', //	Yaqeta Nursing Stations
+  FJ_063: 'FJ_sdLautoka_Yasawa', //	Yasawa-I-Rara Nursing Station
+};
+
+const hierarchyName = 'supplychain_fiji';
```

Comment:
@chris-bes could you please run your eyes over this, I've updated to entity_relations rather than import file which would've messed UNFPA hierarchy.